.github/workflows/ci.yml

```
@@ -0,0 +1,187 @@
+name: Hedera Services CI
+
+on: [push]
```

I would suggest not running CI unless it's a main branch or it's associated with a PR.

```suggestion
on:
  pull_request:
    branches: [ master, release/** ]
  push:
    branches: [ master, release/** ]
    tags: [ v* ]
```